### ISSUE TITLE
[BUG] Text in vi is very hard to see

### ISSUE THREAD

**Issue by jiasli** [Azure/CloudShell#84]
## To Reproduce

```
vi .bashrc
```

## Observed Behavior

Text in `vi` is very hard to see. The color has very low contrast ratio compared to the background.

![image](https://user-images.githubusercontent.com/4003950/111565178-ea159980-87d5-11eb-9d57-2e1178789244.png)

![image](https://user-images.githubusercontent.com/4003950/111564763-40360d00-87d5-11eb-9c24-5c3708e98f42.png)

## Expected behavior

Text in `vi` should have enough contrast ratio compared to the background.

## Is this specific to Cloud Shell?
Yes.

## Interface information

In https://portal.azure.com with Courier New font.

## Additional context

This issue is originally reported at https://github.com/Azure/azure-cli/issues/17325. Workarounds are provided at https://github.com/Azure/azure-cli/issues/17325#issuecomment-801574366.

**Comment by jiasli** [comment]
Some possible solutions:

1. Default the font to Monospace to get bolder font.
2. Make the RGB definition of dark colors lighter. See https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences#text-formatting
3. Allow the user to customize RGB of colors, like the old Windows Console
    ![image](https://user-images.githubusercontent.com/4003950/111566744-b25c2100-87d8-11eb-997f-ffe9166827f2.png)
    and new Windows Terminal
    ![image](https://user-images.githubusercontent.com/4003950/111566805-cc95ff00-87d8-11eb-8d8e-993b51dfe32b.png)

**Comment by edyoung** [comment]
That's a bit odd. My Vi text looks quite different.

![image](https://user-images.githubusercontent.com/45578/111571074-5efbbb80-8763-11eb-9009-3d0d18d641d7.png)

Do you by any chance have a vi color scheme enabled or something like that?

**Comment by jiasli** [comment]
No, because you are already using **Monospace**. Try **Courier New**.

**Comment by mbifeld** [comment]
@jiasli is this still occurring for you? This is what I see now when using Courier New.

<img width="1457" height="940" alt="Image" src="https://github.com/user-attachments/assets/1b31d19d-c551-4ca4-b37e-1f7483f74c3f" />

**Comment by mbifeld** [comment]
Closing based on the above comment. Please reopen / tag me if this issue is still happening for you. Thank you!
